src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -620,6 +615,20 @@
                                        (when (:cycle-focus? props)
                                          (.focus (get-first)))))))))})
 
+(react/defc NoBillingProjectsMessage
+  {:render
+  (fn [{:keys [props]}]
+    [:div {:style {:textAlign "center"}} (str "You must have a billing project associated with your account"
+         " to create a new workspace. To learn how to create a billing project, ")
```

Comment:
I think it is stronger text to have "Learn how to create a billing project" and have that be the link.